### ISSUE TITLE
different results on colab and anaconda on windows

### ISSUE THREAD

**Issue by fizamurtaza** [serengil/deepface#316]
Hello,
I am getting accurate results on google colab but wrong results on anaconda installed on windows. Issue on windows is I am getting wrong results (though getting correct on colab). Although I tried simple installation in different environment.

Can you please help me how I can install it correctly on windows.

**Comment by fizamurtaza** [comment]
![image](https://user-images.githubusercontent.com/25634081/129151828-d2a9de5c-76a0-49c8-9df0-8e4e7f098c5f.png)

**Comment by serengil** [comment]
I am using tf 2.2.0.

Could you delete .pkl file manually and run find function again? btw, I don't understand what is wrong in the command prompt.

**Comment by serengil** [comment]
Here you can find the keras discussion having same trouble: https://github.com/keras-team/keras/issues/4875

Some solved the trouble with uninstalling and reinstalling keras, tf and h5py.

**Comment by serengil** [comment]
closed due to inactivity

**Comment by yuh8** [comment]
@serengil 
This has worked for me for saving and loading weights
use `model.to_json()` to save your architecture config only without weights, and training configuration
save weights using `model.save_weights()`
load model using `model_new = tf.keras.models.model_from_json(model_json)`
compile model using `model_new.compile()`
load weights using `model_new.load_weights()`
After doing above, I was able to get the same level of accuracy

**Comment by fizamurtaza** [comment]
@yuh8, where you done these changes?
